e2e/test_payloads_validator.py

```
@@ -0,0 +1,520 @@
+            if marker in text:
+                found.append(marker.replace(':', ''))
+
+        return found
+
+    @staticmethod
+    def check_homoglyphs(text: str) -> int:
+        """Detecta caracteres homóglifos (cirílico parecido com latino)"""
+        cyrillic_lookalikes = 0
+
+        # Caracteres cirílicos que parecem latinos
+        cyrillic_chars = ['А', 'В', 'С', 'Е', 'Н', 'І', 'О', 'Р', 'Т', 'Х']
```

Ruff (0.14.5)

String contains ambiguous `А` (CYRILLIC CAPITAL LETTER A). Did you mean `A` (LATIN CAPITAL LETTER A)?

(RUF001)